superset/db_engine_specs/base.py

```diff
@@ -697,6 +770,7 @@ def get_cte_query(cls, sql: str) -> Optional[str]:
 
         return None
 
+
```

Comment:
nit: This will also cause a linting error
```suggestion
```

Reply:
Corrected.